src/components/Filters/Filters.stories.tsx

```diff
@@ -123,39 +123,6 @@ export function GroupBy() {
   );
 }
 
-export function GroupByViewAll() {
```

Comment:
This was an example story of the previous "you need to add your own all" that we don't need anyway.